### PR TITLE
Updated text to GoCardless

### PR DESCRIPTION
**Pull request by shall0pass**
I don't have a GoCardless account to update the images.  

Have the images fundamentally changed (besides saying GoCardless instead of Nordigen)?

### CODE REVIEW

**Comment by kyrias** [comment]
The new GoCardless UI is _similar_ but e.g. the menu structure is different, so I could try to take some updated screenshots if you'd like.

File: docs/advanced/bank-sync.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ We are excited to offer this optional bank integration in Actual. Here are a cou
...
 
-### Nordigen Setup
+### GoCardless Setup
 
 **Create SECRET and KEY for Actual**
 
-1. Create an account with Nordigen - https://nordigen.com/
-2. Log into your account dashboard page and select **User secrets** from the left side menu - https://ob.nordigen.com/user-secrets/
+1. Create an account with GoCardless - https://gocardless.com/
+2. Log into your account dashboard page and select **User secrets** from the left side menu - https://ob.gocardless.com/user-secrets/
```
Comment (kyrias) [inline docs/advanced/bank-sync.md:27]:
```suggestion
1. Create an account with GoCardless - https://bankaccountdata.gocardless.com/overview/
2. Log into your account dashboard page and select **User secrets** from the left side menu - https://ob.gocardless.com/user-secrets/
```

The dashboard URL for GoCardless is `https://bankaccountdata.gocardless.com/`, and you sign up in the same place.

File: docs/advanced/bank-sync.md [hunk 2]
```diff
@@ -17,14 +17,14 @@ We are excited to offer this optional bank integration in Actual. Here are a cou
...
 
 ![](/static/img/connecting-your-bank/connecting-your-bank-nordigen-01.png)
```
Comment (kyrias) [inline docs/advanced/bank-sync.md:30]: I can't comment on it apparently (grr, GitHub,) but the actual button is `+ Create New`,  and the top right `Create new Secrets` no longer exists.

**Comment by kyrias** [comment]
Screenshot 2023-07-30 at 18-36-37 Overview Bank Account Data GoCardless
Screenshot 2023-07-30 at 18-38-15 User Secrets Bank Account Data GoCardless
Screen Shot 2023-07-30 at 18 45 34
Screen Shot 2023-07-30 at 18 46 18

**Comment by kyrias** [comment]
The redirect screens have mostly just changed the Nordigen logo to the GoCardless logo.  I tried to get some decent screenshots of them but the problem is that it shows the "company name" on that screen now it doesn't allow for disabling the redirect so I can change the text before taking a screenshot, hm.

Screenshot 2023-07-30 at 18-57-17 Grant permission Nordigen

**Comment by shall0pass** [comment]
I've updated the images based on what you were able to get.  Do they feel 'washed out' to you?  There's more contrast on the dark mode.  I don't think we have borders around any other images in the docs, but maybe we should on these?

File: static/img/connecting-your-bank/connecting-your-bank-gocardless-02.png:Zone.Identifier
Comment (kyrias) [inline static/img/connecting-your-bank/connecting-your-bank-gocardless-02.png:Zone.Identifier:1]: These appear to be some kind of Windows streams stuff.

**Comment by kyrias** [comment]
> I've updated the images based on what you were able to get. Do they feel 'washed out' to you? There's more contrast on the dark mode. I don't think we have borders around any other images in the docs, but maybe we should on these?

You don't appear to have changed the page to actually reference the new ones?  :)

File: static/img/connecting-your-bank/connecting-your-bank-gocardless-02.png:Zone.Identifier
Comment (kyrias) [reply static/img/connecting-your-bank/connecting-your-bank-gocardless-02.png:Zone.Identifier:1]: Did you perhaps copy these into WSL2 as per https://cloudbytes.dev/snippets/wsl2-find-and-delete-zoneidentifier-files?

Comment (shall0pass) [reply static/img/connecting-your-bank/connecting-your-bank-gocardless-02.png:Zone.Identifier:1]: I did.  I noticed it last night but haven't gotten around to deleting them yet.

**Comment by shall0pass** [comment]
> > I've updated the images based on what you were able to get. Do they feel 'washed out' to you? There's more contrast on the dark mode. I don't think we have borders around any other images in the docs, but maybe we should on these?
> 
> You don't appear to have changed the page to actually reference the new ones? :)

Weird.  I guess the commit didn't take.  I was viewing the page locally before I pushed, so I was definitely updating the links. Later today :)

File: docs/advanced/bank-sync.md [hunk 3]
```diff
@@ -17,45 +17,45 @@ We are excited to offer this optional bank integration in Actual. Here are a cou
...
 
-### Nordigen Setup
+### GoCardless Setup
 
 **Create SECRET and KEY for Actual**
 
-1. Create an account with Nordigen - https://nordigen.com/
-2. Log into your account dashboard page and select **User secrets** from the left side menu - https://ob.nordigen.com/user-secrets/
+1. Create an account with GoCardless - https://bankaccountdata.gocardless.com/overview/
+2. Log into your account dashboard page and select **User secrets** from the left side menu - https://ob.gocardless.com/user-secrets/
```
Comment (kyrias) [inline docs/advanced/bank-sync.md:27]:
```suggestion
2. Log into your account dashboard at the same URL and select **User secrets** from the left side menu
```

**Comment by shall0pass** [comment]
It's this in a 'good enough' state to merge?  There are still some screenshots that mention nordigen on the page.

**Review by kyrias — APPROVED** [review]

**Review by joel-jeremy — APPROVED** [review]

**Comment by drgndrp** [comment]
I recently tried to set up GoCardless for a friend and i believe they changed their business model, you now have to get your app approved to get an api key :/

**Pull request merged**